django_site/settings.py

```diff
@@ -246,12 +246,7 @@
     'djangocms_video': 'djangocms_video.migrations_django',
 }
 
-
-def get_module_name():
-    return os.getenv('MODULE_NAME')
-
-
-AIMMO_GAME_SERVER_URL_FUNCTION = lambda game: (os.getenv('MODULE_NAME') + '-aimmo.codeforlife.education',
+AIMMO_GAME_SERVER_URL_FUNCTION = lambda game: (os.getenv('DJANGO_MODULE_NAME') + '-aimmo.codeforlife.education',
```

Comment:
In a failure scenario on DJANGO's side that isn't testable here, could we have a default fallback for `os.getenv`?

Reply:
Scrap this. We need `None` failure here.